### PR TITLE
Ran/login app form progress

### PR DESCRIPTION
**Pull request by stormcrow85**
**NOTE:** PLEASE REVIEW THIS AFTER Ran/login app bootstrap IS IN

### CODE REVIEW

File: demo/loginApp/index.html [hunk 1]
```diff
@@ -3,44 +3,26 @@
...
-<script src="plivo/dist/plivo.min.js"></script>
 <script src="angular-messages/angular-messages.js"></script>
 <script src="angular-material/angular-material.js"></script>
-<script src="angular-ui-router/release/angular-ui-router.js"></script>
 <script src="infra/dist/main.js"></script>
-<script src="ng-tags-input/ng-tags-input.js"></script>
-<script src="Chart.js/Chart.js"></script>
-<script src="angular-chart.js/dist/angular-chart.js"></script>
-<script src="v-accordion/dist/v-accordion.js"></script>
+<script src="angular-svg-round-progressbar/build/roundProgress.js"></script>
```
Comment (assafshp) [inline demo/loginApp/index.html]: do we need it ? roundprogress?

File: dist/imageZoomer/imageZoomer.js [hunk 2]
```diff
@@ -0,0 +1,202 @@
+(function (angular) {
+    'use strict';
+
+    angular.module('znk.infra-web-app.imageZoomer', [
```
Comment (assafshp) [inline dist/imageZoomer/imageZoomer.js]: why do we need imageZOOMER

File: dist/imageZoomer/main.css [hunk 3]
```diff
@@ -0,0 +1,57 @@
+/* fonts size */
+/* general */
```
Comment (assafshp) [inline dist/imageZoomer/main.css]: ?

File: dist/loginApp/loginApp.js [hunk 4]
```diff
@@ -90,8 +90,8 @@
                         if (search.state) {
                             scope.changeCurrentForm(search.state);
                         }
-                        $location.search('app', null);
-                        $location.search('state', null);
+                        // $location.search('app', null);
```
Comment (assafshp) [inline dist/loginApp/loginApp.js]: NO NEED for that ?

**Comment by assafshp** [comment]
Please merge zinkerz dev to see only updates

File: dist/imageZoomer/imageZoomer.js (hunk 2 above)
Comment (stormcrow85) [inline dist/imageZoomer/imageZoomer.js]: i didn't add this, it was added when merging & dist

File: dist/imageZoomer/main.css (hunk 3 above)
Comment (stormcrow85) [inline dist/imageZoomer/main.css]: no idea, didn't write this

File: dist/loginApp/loginApp.js (hunk 4 above)
Comment (stormcrow85) [inline dist/loginApp/loginApp.js]: in the meantime no, but it was a quickfix, in the future i guess you would want to remove that url parameters, so i left the comment there

File: demo/loginApp/index.html (hunk 1 above)
Comment (stormcrow85) [inline demo/loginApp/index.html]: not anymore, removed

File: dist/feedback/main.css [hunk 5]
```diff
@@ -1,256 +0,0 @@
-/*border color*/
```
Comment (assafshp) [inline dist/feedback/main.css]: why did you delete it?
